### PR TITLE
iOS Release v9.1.0

### PR DESCRIPTION
**Pull request by MYelfimova**
update release notes for KidozSDK 9.1.0 and KidozIronSourceAdapter 1.3.0

### CODE REVIEW

File: Kidoz Direct/iOS/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 9.1.0 - Mar 14, 2024
+
+* Changed the min supported iOS version to iOS 11
```
Comment (yarden-rosenberg) [inline Kidoz Direct/iOS/CHANGELOG.md:5]: Updated the min supported iOS version to iOS 11

and move it to last place

File: Kidoz Direct/iOS/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+* Added SKOverlay for performance improvement
+* Added Privacy Manifest and Signature
+
 ## 9.0.1 - Oct 4, 2023
 
-* Rebuilt to support Xcode 14.2 
+* Updated min supported XCode version to 14.2
```
Comment (yarden-rosenberg) [inline Kidoz Direct/iOS/CHANGELOG.md:11]: It is actual min 14.1 as it is the same clang

File: Kidoz Direct/iOS/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # Kidoz iOS Mobile SDK
 
-Kidoz SDK is compatible with iOS 10.0 and above and is written in Objective-C and Swift.<BR>
-Minimum supported Xcode version is 14.1.
+Kidoz SDK is compatible with iOS 11.0 and above and is written in Objective-C and Swift.<BR>
+Minimum supported Xcode version is 14.2.
```
Comment (yarden-rosenberg) [inline Kidoz Direct/iOS/README.md:4]: Needs to stay Minimum supported Xcode version is 14.1.

File: Mediation/AdMob Adapter/iOS/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # Kidoz Admob iOS Adapter
 The Kidoz Admob adapter was built and tested with Google-Mobile-Ads-SDK v10.8.0.<BR>
-The minimum supported Xcode version is 14.1.<BR><BR>
+The minimum supported Xcode version is 14.2.<BR><BR>
```
Comment (yarden-rosenberg) [inline Mediation/AdMob Adapter/iOS/README.md:3]: Again The minimum supported Xcode version is 14.1 is better

File: Kidoz Direct/iOS/SampleApp/Pods/KidozSDK/KidozSDK/KidozSDK.xcframework/_CodeSignature/CodeDirectory
[the remark this answers is not in the document]
Comment (MYelfimova) [reply Kidoz Direct/iOS/SampleApp/Pods/KidozSDK/KidozSDK/KidozSDK.xcframework/_CodeSignature/CodeDirectory:1]: why to remove .xcframework? 
It was installed via 'pod install' as a dependency and will be easier for publishers if the xcframework will be already there, no? @yarden-rosenberg

**Review by yarden-rosenberg — APPROVED** [review]

**Pull request merged**